.github/workflows/conformance.yml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   build:
-    if: ${{ github.repository == 'panva/node-oidc-provider' || github.event_name == 'workflow_dispatch' }}
+    if: ${{ github.repository == 'logto-io/node-oidc-provider' || github.event_name == 'workflow_dispatch' }}
     uses: panva/.github/.github/workflows/build-conformance-suite.yml@main
```

Comment:
Should we also replace this `panva` with `logto-io`?

Reply:
i think it's fine since we can keep the upstream update using this

Reply:
aligned offline